src/clients/python/examples/simple_grpc_shm_string_client.py

```diff
@@ -36,6 +36,30 @@
 
 FLAGS = None
 
+
+def get_number_of_bytes_for_npobject(tensor_value):
```

Comment:
This is the function that we can potentially include in the Triton Python client.

Reply:
Do you mean `tritonclient.utils`?

Reply:
Yes

Reply:
I agree. But then we should change the name to serialized_byte_size and make it generic for all both np.object and non-np.object type. If there a reason we should have this for a specific tensor value instead of a tenor of values?

Reply:
I changed the name to `serialized_byte_size` but I guess because of the name, it shouldn't work for other types. I have raised an exception if a different type is provided.